### ISSUE TITLE
with_items: php_packages is not getting expanded

### ISSUE THREAD

**Issue by wojtha** [geerlingguy/ansible-role-php#185]
I have to do this to get this playbook working

```diff
- with_items: php_packages
+ with_items: "{{ php_packages }}"
```

Otherwise the playbook was failing with this message:

```
TASK [geerlingguy.php : Ensure PHP packages are installed.] ********************
failed: [192.168.2.3] (item=[u'php_packages']) => {"changed": false, "failed": true, "item": ["php_packages"], "msg": "No package matching 'php_packages' found available, installed or updated", "rc": 126, "results": ["No package matching 'php_packages' found available, installed or updated"]}
```
What could be wrong? It should be working according to e.g. this issue: https://github.com/ansible/ansible/issues/2876

This is my environment:

- Ansible 2.2.2.0
- Python 2.7.13
- Vagrant 1.8.5
- Darwin 15.6.0 Darwin Kernel Version 15.6.0: Fri Feb 17 10:21:18 PST 2017; root:xnu-3248.60.11.4.1~1/RELEASE_X86_64 x86_64

**Comment by geerlingguy** [comment]
Interesting... the tests pass, and in my local environment I can run it with Ansible 2.2.1.0 at least. This could be a regression in Ansible 2.2.2.0, maybe?

Otherwise, are you sure it's using your `ansible` provisioner on the host and not `ansible_local` within the VM, where an older version of Ansible might be installed?

**Comment by oxyc** [comment]
I'm confused, your diff indicates it should be `{{ php_packages }}`, yes?

Which it is:
- https://github.com/geerlingguy/ansible-role-php/blob/66bfceb596d64c8b6d0c0307194e2bcbde170455/tasks/setup-RedHat.yml#L7
- https://github.com/geerlingguy/ansible-role-php/blob/08c948d24f69a71058c9f0ad73406a1298cd9747/tasks/setup-Debian.yml#L10

**Comment by geerlingguy** [comment]
@wojtha - Hmm, maybe you're running an older version of Drupal VM? It was probably formatted like you had originally in an older version.

**Comment by wojtha** [comment]
@geerlingguy nope I was reading your book Jeff (good one, thanks!) and I was trying out some stuff. So I installed the role via `ansible-galaxy` yesterday or maybe few days ago, I started to read it last Friday and I wasn't using ansible before so the role wasn't certainly there. It happened while I was provisioning Vagrant boxes in [lamp-infrastructure](https://github.com/geerlingguy/ansible-for-devops/tree/master/lamp-infrastructure) chapter.

**Comment by wojtha** [comment]
@oxyc @geerlingguy hmm I tried to run `sudo ansible-galaxy install geerlingguy.php -f` and the `"{{ php_packages }}"`is there now. Unfortunately I didn't look at the file timestamp or version number before I did it.

So it is probable that maybe I have this role installed without knowing about it. Maybe I was trying Drupal VM (as I am former developer, even the core maintainer) long time ago so it seems that it was problem on my side. So sorry for bothering to both of you!
